deploy/helm/sumologic/README.md

```diff
@@ -127,6 +127,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `sumologic.pullSecrets`                                                          | Optional list of secrets that will be used for pulling images for Sumo Logic's deployments and statefulsets.                                                                                                                                                                                                                                                         | `Nil`                                                                                                                                                                                |
 | `sumologic.otelcolImage.repository`                                              | Default image repository for OpenTelemetry Collector. This can be overridden for specific components.                                                                                                                                                                                                                                                                | `public.ecr.aws/sumologic/sumologic-otel-collector`                                                                                                                                  |
 | `sumologic.otelcolImage.tag`                                                     | Default image tag for OpenTelemetry Collector. This can be overridden for specific components.                                                                                                                                                                                                                                                                       | `0.76.1-sumo-0`                                                                                                                                                                      |
+| `sumologic.otelcolImage.addFipsSuffix`                                           | Add a -fips suffix to all image tags. See docs/security-best-practices.md for more information.                                                                                                                                                                                                                                                                      | `false`                                                                                                                                                                              |
```

Comment:
```suggestion
| `sumologic.otelcolImage.addFipsSuffix`                                           | Add a `-fips` suffix to all image tags. See docs/security-best-practices.md for more information.                                                                                                                                                                                                                                                                    | `false`                                                                                                                                                                              |
```

Reply:
also please link `docs/security-best-practices.md`

Reply:
This is how we refer to all Chart docs in the README, I assume because there's no way to render the links correctly in ArtifactHub.

Reply:
`Extra processors for container logs. See [/docs/collecting-container-logs.md](/docs/collecting-container-logs.md) for details.`

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/2277f7fb1977369296214566840d52dcbabc2b2c/deploy/helm/sumologic/README.md?plain=1#LL63C72-L63C72

Reply:
Ok, done.